### PR TITLE
feat: make ctrl+enter in flows run a preview on the current flow

### LINKED ISSUE #643: Notebooks: I can preview a notebook using keyboard shortcuts
As a InfluxDB Cloud user, I can preview my notebook using a keyboard shortcut so that I can avoid needing to scroll bath to the top of the page to preview my changes.

Scenario:
Given: I have a notebook open with cells in it
When: I type cmd + enter on my keyboard
Then: the notebook is previewed

- Technical Notes
  - This is the same keyboard shortcut used in the script editor
- Design Notes
- Clockface Component
- Mockup

### PR DESCRIPTION
**Pull request by joelfried**
Makes ctrl+enter run a preview when looking at a flow.

Added e2e test coverage for this case.

Closes #643

### CODE REVIEW

File: src/flows/components/Flow.tsx [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
+  const {queryAll} = useContext(FlowQueryContext)
+
+  const runPreview = event => {
+    if (event.key === 'Enter' && event.ctrlKey) {
+      if (runMode === RunMode.Preview) {
+        queryAll()
+      } else {
+        setRunMode(RunMode.Preview)
+        queryAll()
+        setRunMode(RunMode.Run)
```
Comment (Asalem1) [inline src/flows/components/Flow.tsx:21]: not sure about this part

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 49 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1209)

File: src/flows/components/Flow.tsx [hunk 2]
```diff
@@ -1,15 +1,39 @@
...
+      }
+    }
+  }
+
+  React.useEffect(() => {
```
Comment (drdelambre) [inline src/flows/components/Flow.tsx:26]: import `useEffect` like you did for `useContext`

**Comment by drdelambre** [comment]
it'd be great to see this as it's own component, included at the same level as the `Page` component in the `FlowContainer` component

File: src/flows/components/Flow.tsx (hunk 1 above)
Comment (joelfried) [reply src/flows/components/Flow.tsx:21]: I'm open to another way to do it but as far as I could tell queryAll() reads the current state of RunMode so I needed it to be Preview when the queryAll() fires.  I thought about changing RunMode to Preview and leaving it but wasn't sure users would want that behavior.

**Review by Asalem1 — APPROVED** [review]

**Pull request merged**